src/azul/health.py

```diff
@@ -241,3 +270,45 @@ def update_cache(self) -> None:
     @classproperty
     def all_keys(cls) -> Set[str]:
         return {p.key for p in cls.all_properties}
+
+    def archive_fail_messages(self, remaining_context_time):
+        client = boto3.client('dynamodb')
+        fail_queue = boto3.resource('sqs').get_queue_by_name(QueueName=config.fail_queue_name)
+        while remaining_context_time.get() >= self.receive_message_wait_time + 2:  # 2 second margin for db actions
+            received_messages = fail_queue.receive_messages(AttributeNames=['All'],
+                                                            MaxNumberOfMessages=10,
+                                                            WaitTimeSeconds=self.receive_message_wait_time,
+                                                            VisibilityTimeout=300)
+            if received_messages:
+                message_batch = [
+                    {
+                        'PutRequest': {
+                            'Item': {
+                                'MessageType': {
+                                    'S': 'bundle' if 'notification' in json.loads(message.body).keys() else 'other'
```

Comment:
Since we eliminated tokens this conditional can be removed. All messages in this queue will be notifications.

But in https://github.com/DataBiosphere/azul/issues/1593 we will be adding failed tallies to this queue so it makes sense to hardcode this to `notification` in this PR.